### ISSUE TITLE
Dependencies in Cargo.toml are local

### ISSUE THREAD

**Issue by m-hilgendorf** [Connicpu/direct2d-rs#9]
I'm chasing down an error that requires me to compile this crate separately and I can't build this crate without reverting before [this commit](https://github.com/Connicpu/direct2d-rs/commit/99a4c72e4744ee8fe4b4ab0d1ae2dcdacbe3c993). 

I'm guessing that you're working on stuff locally so it's not a big deal, but it would be nice if references to local dependencies were in a separate fork before being merged with the master.

**Comment by Connicpu** [comment]
Ah right, I forgot that strategy breaks git dependencies dangit. I can push another alpha version later today if you'd like!

**Comment by m-hilgendorf** [comment]
Whenever you have time! Like I said it's not a huge deal, I'm guessing you're working on some big changes so it makes sense why it's there.

**Comment by Connicpu** [comment]
Yeah :) Most of the major changes are already done, the rest is mostly just filling in some more APIs, but I've just published all of the latest versions to crates.io now if you'd like to try them!

**Comment by frabert** [comment]
Hi! Any update on this? I'd like to extend some of the API, but I'm not sure how to handle the dependencies.
